content-api/main.py

```diff
@@ -20,6 +20,33 @@
 
 from resources import methods
 
+from opentelemetry import propagate
+from opentelemetry import trace
+
+from opentelemetry.exporter.cloud_trace import CloudTraceSpanExporter
+from opentelemetry.instrumentation.flask import FlaskInstrumentor
+from opentelemetry.propagators.cloud_trace_propagator import CloudTraceFormatPropagator
+from opentelemetry.propagators.composite import CompositePropagator
+from opentelemetry.sdk.trace.export import BatchSpanProcessor
+from opentelemetry.sdk.trace import TracerProvider
+
+provider = TracerProvider()
+processor = BatchSpanProcessor(CloudTraceSpanExporter())
+provider.add_span_processor(processor)
+
+# Add the X-Cloud-Trace-Context propagator.
```

Comment:
Does this service only get the X-CTC header? I think if you switch the order of the arguments in your composite propagator it will try the defaults (tracecontext, baggage) first, and fall back to X-CTC if necessary.  A minor detail,  but you might check if there are multiple headers being sent in and prefer to propagate tracecontext instead.

Reply:
Good catch, Drew!
I've also updated this to use the new OneWay propagator, which will read XCTC if present, but injects no headers on its own, reinforcing the use of `traceparent`.